contributing-guides/maintainers-guide.md

```diff
@@ -93,6 +97,8 @@ for the behavior expected of tldr-pages maintainers.
   is that if there are more "dirty" commits than "clean" commits,
   then prefer squash, else do a rebase.
 
+- It is suggested to preserve the coauthored by message when cleaning the body of the squashed commit message unless the change done was trivial.
```

Comment:
I think we should establish a convention for commit messages. The `git log` of this repo is kind of a mess. A lot of stuff is repeated multiple times.

````suggestion
- It is suggested to clean up the commit message when merging a PR. For small commits, use:
  ```
  page-name: short description of the change

  Co-authored-by: ...
  ```
  if you think a more descriptive message is needed, use asterisks:
  ```
  page-name: short description of the change

  * some more information
  * ...

  ---------

  Co-authored-by: ...
  ```

- It is suggested to preserve the `Co-authored-by` message when cleaning the body of a squashed commit message unless the change done was trivial.
````